src/client/components/SearchFilterComponent/SearchFilter.styles.css

```diff
@@ -0,0 +1,304 @@
+    border-width: 0.1rem;
+    border-color: #C4C4C4;
+    }
+
+.container  h2 img{
+   position:relative;
+   bottom: 0rem;
+   width: 10px;
+   left:14rem;
+   }
+
+hr{
+   width: 22rem;
+   color: #c9bcbc;
+   }
+.container .input {
+   background-color: #ffffff;
+   width: 320px;
+   height: 103px;
+}
+.container .input input{
+  margin-top: 0.7rem;
+  margin-left: 1.5rem;
+}
+.container .input b{
+  font-family: roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  margin-top: 0.7rem;
+  margin-left: 0.2rem;
+  
+}
+
+/* Date posted */
+
+.container .posted{
+  background-color: #ffffff;
+  width: 320px;
+  height: 53px;
+  margin-top: 0rem;
+}
+.container  h3{
+  margin-top: 0.7rem;
+  margin-left: 1.5rem;
+  font-family: 'Times New Roman', Times, serif; 
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+    line-height: 21px;
+    color:rgb(0, 0, 0);
+    width: 340px;
+    height: 40px;
+    background-color: #ffffff;
+    padding-top: 0.5rem;
+    padding-left: 2.5rem;
+    position: relative;
+    right: 2.1rem;
+    border-style: solid;
+    border-width: 0.1rem;
+    border-color: #C4C4C4;
+    }
+    .container  h3 img{
+     position:relative;
+     bottom: 0rem;
+     width: 10px;
+     left:10rem;
+    }
+    .container .inputDate {
+     background-color: #ffffff;
+     width: 320px;
+     height: 123px;
+    }
+    .container .inputDate input{
+     margin-top: 0.7rem;
+     margin-left: 1.5rem;
+    }
+
+    .container .inputDate b{
+     font-family: roboto;
+     font-style: normal;
+     font-weight: normal;
+     font-size: 15px;
+     line-height: 21px;
+     margin-top: 0.7rem;
+     margin-left: 0.2rem;
+    }
+
+    /* minimum evaluation */
+
+    .container .minimum{
+     background-color: #ffffff;
+     width: 320px;
+     height: 53px;
+     margin-top: 0rem;  
+    }
+
+    .container  h4{
+     margin-top: 0.7rem;
+     margin-left: 1.5rem;
+        font-family: 'Times New Roman', Times, serif;
+        font-style: normal;
+        font-weight: bold;
+        font-size: 20px;
+        line-height: 21px;
+        color:rgb(0, 0, 0);
+        width: 340px;
+        height: 40px;
+        background-color: #ffffff;
+        padding-top: 0.5rem;
+        padding-left: 2.5rem;
+        position: relative;
+        right: 2.1rem;
+        border-style: solid;
+        border-width: 0.1rem;
+        border-color: #C4C4C4;
+        }
+
+        .container  h4 img{
+            position:relative;
+            bottom: 0rem;
+            width: 10px;
+            left:5rem;
+        }
+
+        .container .inputMinimum {
+            background-color: #ffffff;
+            width: 320px;
+            height: 163px;
+        }
+       
+        .container .inputMinimum input{
+            margin-top: 0.7rem;
+            margin-left: 1.5rem;
+        }
+
+        .container .inputMinimum b{
+            font-family: roboto;
+            font-style: normal;
```

Comment:
```suggestion

```

Reply:
here is the same as in the mockup

Reply:
If you go to Figma, you can click on an element and on the right side, see which properties an element has, but very important: Never copy exactly everything that appears in Figma, see that more as a guide. One does not have to give a line height, position: absolute, font-weight:normal, left, right, top, bottom... etc. every single time. 
Overall, I would say, must of the properties that appear in Figma are useless for us. 
What we do want to check is fx. font-family (but if it has been set for the whole document, it does not have to be specified every single time), font-weight in case is other than normal, color, or maybe font-size  

![Screenshot (2522)_LI](https://user-images.githubusercontent.com/36211640/107793028-23319880-6d56-11eb-9e7d-3a36b1dd0343.jpg)
![Screenshot (2521)_LI](https://user-images.githubusercontent.com/36211640/107793032-23ca2f00-6d56-11eb-8e1c-0adbde094448.jpg)

Reply:
wich is the correct font-family ?

Reply:
`Roboto `for must cases, In other cases it `Jost`. Under typography in Figma, you can see it where it says Font. In this component, I believe you should just use Roboto.

Reply:
And you don't have to specify the font `Roboto `every time, in some cases when the same class is used in multiple components, the fonts might overwrite. But you can test by not specifying the font-family and check which one is rendered.